### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by Kotochleb**

### CODE REVIEW

File: .github/workflows/catkin_test_build.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: ROS1 test build
+
+on:
+  push:
+    branches:
+      - 'ros1-devel'
```
Comment (macstepien) [inline .github/workflows/catkin_test_build.yaml]: Probably it would be better to run it on branch `ros1` and on PRs

File: README.md [hunk 2]
```diff
@@ -1,52 +1,63 @@
...
+## Parameters
 
-Predefined wheels:
-- `WH01.yaml` - off-road skid drive wheels. Panther standard wheels
+Arguments passed to the [panther.urdf.xacro](./panther_description/urdf/panther.urdf.xacro) are the same as parameters of [panther_macro.urdf.xacro](./panther_description/urdf/panther_macro.urdf.xacro). Thus, this section covers both of them.
 
-- `WH02.yaml` - mecanum wheels 
 
-- `WH04.yaml` - small skid drive wheels
+- `use_sim` *(default: false)* - Changes between *ros2_control* for simulation and hardware **[TO BE IMPLEMENTED]**.
```
Comment (macstepien) [inline README.md]:
```suggestion
- `use_sim` *(default: false)* - Changes between *ros_control* for simulation and hardware **[TO BE IMPLEMENTED]**.
```

File: README.md [hunk 3]
```diff
@@ -1,52 +1,63 @@
...
+It is not present in the [panther_macro.urdf.xacro](./panther_description/urdf/panther_macro.urdf.xacro) since base of a robot does not have any sensors that can be accelerated with GPU.
 
-Predefined sensors are:
-- Orbbec Astra
-- Slamtec RPLIDAR S1
-- Velodyne Puck
-- Ouster OS1 32
+Parameter `wheel_config_path` allows using non-standard wheels with Panther robot without modifying URDF file. Syntax is following:
+- `wheel_radius` - wheel radius.
+- `wheel_separation` - separation of wheels alongside *y* axis.
```
Comment (macstepien) [inline README.md]: Maybe also add units here (they are specified in other parameters)?

File: package.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>panther_description</name>
+  <version>0.1.0</version>
+  <description>Semantir description of Huasrion Panther robot</description>
```
Comment (macstepien) [inline package.xml]:
```suggestion
  <description>Semantic description of Huasrion Panther robot</description>
```

File: urdf/body.urdf.xacro [hunk 5]
```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <!-- body defining macro -->
+  <xacro:macro name="body" params="
+    wheel_radius simulation_engine
+    imu_pos_x imu_pos_y imu_pos_z
+    imu_rot_r imu_rot_p imu_rot_y" >
+
+    <link name="base_link"></link>
```
Comment (macstepien) [inline urdf/body.urdf.xacro]:
```suggestion
    <link name="base_link" /> 
```

File: urdf/body.urdf.xacro [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+    </link>
+
+    <joint name="body_link_to_imu_link_joint" type="fixed">
+      <origin xyz="${imu_pos_x} ${imu_pos_y} ${imu_pos_z}"
+              rpy="${imu_rot_r} ${imu_rot_p} ${imu_rot_y}" />
+      <parent link="body_link" />
+      <child link="imu_link" />
+    </joint>
+
+    <link name="imu_link"></link>
```
Comment (macstepien) [inline urdf/body.urdf.xacro]: I see that there is more of closing `</link>`, I prefer to close one liners with ` />`

File: urdf/panther.urdf [hunk 7]
```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" ?>
+<!-- =================================================================================== -->
+<!-- |    This document was autogenerated by xacro from panther.urdf.xacro             | -->
+<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
+<!-- =================================================================================== -->
```
Comment (macstepien) [inline urdf/panther.urdf]: Is this file necessary?

File: urdf/panther_macro.urdf.xacro [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+          <plugin name='skid_steer_drive_controller' filename='libgazebo_ros_skid_steer_drive.so'>
+            <updateRate>20.0</updateRate>
+            <robotBaseFrame>base_link</robotBaseFrame>
+            <wheelSeparation>${wheel_config['wheel_separation']}</wheelSeparation>
+            <wheelDiameter>${wheel_config['wheel_radius']*2.0}</wheelDiameter>
+            <torque>34.5</torque>
+            <leftFrontJoint>fl_joint</leftFrontJoint>
+            <rightFrontJoint>fr_joint</rightFrontJoint>
+            <leftRearJoint>rl_joint</leftRearJoint>
+            <rightRearJoint>rr_joint</rightRearJoint>
```
Comment (macstepien) [inline urdf/panther_macro.urdf.xacro]: Sorry that I missed it in ROS2 PR, but wouldn't it be better to add `wheel` to joint name (e.g. `fl_wheel_joint`)? I think that  only position prefix doesn't necessarily tell that it will be a wheel.

File: urdf/wheel.urdf.xacro [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+      <parent link="body_link" />
+      <child link="${prefix}_wheel_link" />
+      <origin xyz="${x} ${y} 0.0" rpy="0.0 0.0 0.0" />
+      <axis xyz="0.0 1.0 0.0" />
+
+      <limit effort="34.5"
+             velocity="2.0" />
+
+      <dynamics damping="0.01"
+                friction="0.01"/>
```
Comment (macstepien) [inline urdf/wheel.urdf.xacro]:
```suggestion
      <limit effort="34.5" velocity="2.0" />
      <dynamics damping="0.01" friction="0.01" />
```

**Review by macstepien — CHANGES_REQUESTED** [review]

File: .github/workflows/catkin_test_build.yaml (hunk 1 above)
Comment (Kotochleb) [reply .github/workflows/catkin_test_build.yaml]: Done

File: README.md (hunk 2 above)
Comment (Kotochleb) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (Kotochleb) [reply README.md]: Done

File: urdf/body.urdf.xacro (hunk 6 above)
Comment (Kotochleb) [reply urdf/body.urdf.xacro]: Done

File: urdf/panther.urdf (hunk 7 above)
Comment (Kotochleb) [reply urdf/panther.urdf]: Removed

File: urdf/panther_macro.urdf.xacro (hunk 8 above)
Comment (Kotochleb) [reply urdf/panther_macro.urdf.xacro]: Done

**Review by macstepien — APPROVED** [review]
